Blockstack/Classes/Blockstack.swift

```diff
@@ -9,23 +9,27 @@ import Foundation
 
 open class Blockstack {
     
-    private var instance: BlockstackInstance?
+    private var authenticationModule: BlockstackAuthentication
+    private var storageModule: BlockstackStorage
     
-    private static var sharedBlockstack: Blockstack = {
+    open static var shared: Blockstack = {
         let blockstack = Blockstack()
         return blockstack
     }()
     
-    open class func sharedInstance() -> BlockstackInstance {
-        return sharedBlockstack.getInstance()
+    init() {
+        authenticationModule = BlockstackAuthentication()
+        storageModule = BlockstackStorage()
```

Comment:
`Blockstack` class allows us to easily configure and inject all the modules, which helps with configuration and unit testing.